java/client/src/org/openqa/selenium/remote/tracing/EventAttributeValue.java

```diff
@@ -44,6 +48,26 @@ public EventAttributeValue(boolean value) {
     this.type = Type.BOOLEAN;
   }
 
+  public EventAttributeValue(String[] value) {
+    this.stringArrayValue = value;
+    this.type = Type.STRING_ARRAY;
+  }
+
+  public EventAttributeValue(Long[] value) {
```

Comment:
Again, prefer primitive types `long` to `Long`. This avoids the need for null checks in your own code.